assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -209,8 +215,9 @@ const baseResolvers = {
 		// Only fetch accounts if there are none in the store.
 		if ( existingAccounts === undefined ) {
 			yield tagActions.waitForExistingTag();
-			const existingTag = registry.select( STORE_NAME ).getExistingTag();
+
 			let existingTagPermission;
+			const existingTag = registry.select( STORE_NAME ).getExistingTag();
```

Comment:
These changes seem unrelated?

However, actually this resolver is missing necessary extra calls to `receiveError`:
* `getAccounts()` doesn't receive arguments, but its fetch store...
    * has a different name _and_
    * receives arguments
* This means any error will be keyed incorrectly by default.

We will need to check if there's an `error` below, and if so store it in `receiveError( error, 'getAccounts', [] )`.

Reply:
Yep, it's unrelated, i have reverted it and added error handling for getAccounts resolver..